string application
int8 priority
string da_name
---
bool status 